Library/Homebrew/cmd/test-bot.rb

```diff
@@ -572,6 +582,14 @@ def formulae
       changed_formulae + unchanged_formulae
     end
 
+    def head_only?(formula)
+      formula.head && formula.devel.nil? && formula.stable.nil? && tap = "homebrew/homebrew-head-only"
```

Comment:
That's with ` tap == "homebrew/homebrew-head-only" ` rather than ` tap = "homebrew/homebrew-head-only" `